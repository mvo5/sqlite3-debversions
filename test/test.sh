#!/bin/sh

EXPECTED=$(cat test.expected)
res=$(sqlite3 < test.sql)
if [ "$res" != "$EXPECTED" ]; then
    echo "test failed, got:"
    echo "$res"
    echo "expected:"
    echo "$EXPECTED"
    exit 1
fi
