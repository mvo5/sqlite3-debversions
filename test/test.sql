.load "../debversions.so"
create table versions (version STRING);
insert into versions values("1.0-1~lucid1");
insert into versions values("1.0-1");
SELECT version FROM versions ORDER BY version COLLATE debversion_compare;


