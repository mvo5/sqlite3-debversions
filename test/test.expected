1.0-1~lucid1
1.0-1

